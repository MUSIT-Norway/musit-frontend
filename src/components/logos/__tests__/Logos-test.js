import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';
import NotFound from '../Logos';

describe('Logos', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <NotFound />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
