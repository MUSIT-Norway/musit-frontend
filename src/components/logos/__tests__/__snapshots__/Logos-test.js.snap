// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logos should match snapshot 1`] = `
<div
  className="logos"
>
  <img
    alt="logo"
    src="UiS_nor_color_rgb.png"
    style={
      Object {
        "height": "60px",
      }
    }
  />
  <img
    alt="logo"
    src="UiO_SAMARB_rgb_gjennomsiktig.png"
    style={
      Object {
        "height": "60px",
      }
    }
  />
  <img
    alt="logo"
    src="UiT_samarbeidslogo_bokmal_300ppi.png"
    style={
      Object {
        "height": "60px",
      }
    }
  />
  <img
    alt="logo"
    src="UiBmerke_grayscale.png"
    style={
      Object {
        "height": "60px",
      }
    }
  />
  <img
    alt="logo"
    src="ntnu_u-slagord.png"
    style={
      Object {
        "height": "60px",
      }
    }
  />
</div>
`;
